cli/src/registry/control-bar/control-bar.tsx

```diff
@@ -23,9 +23,17 @@ export interface ControlBarProps extends React.HTMLAttributes<HTMLDivElement> {
 const ControlBar = React.forwardRef<HTMLDivElement, ControlBarProps>(
   ({ className, contextKey, hotkey = "mod+k", ...props }, ref) => {
     const [open, setOpen] = React.useState(false);
+    const [isMac, setIsMac] = React.useState(false);
     const { thread } = useTambo();
     const scrollContainerRef = useRef<HTMLDivElement>(null);
 
+    useEffect(() => {
```

Comment:
see my other changes for isMac - we shouldn't be using useEffect here, it is an extra render for something we can just get at initial render time